src/test/scala/Tests.scala

```diff
@@ -0,0 +1,61 @@
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers._
+
+
+class Tests extends AnyFlatSpec {
+
+
+
+  "iterator" should "return header and line" in{
```

Comment:
You are actually testing the Scala Scala code and not our code, you need to test our code